enterprise/frontend/test/metabase-enterprise/audit/auditing.cy.spec.js

```diff
@@ -141,29 +114,29 @@ describeWithToken("audit > auditing", () => {
     it("should load the All Members tab", () => {
       cy.visit("/admin/audit/members/all");
 
-      all_users.forEach(user => {
-        cy.findByText(user.first_name + " " + user.last_name);
+      Object.values(USERS).forEach(({ first_name, last_name }) => {
+        cy.findByText(`${first_name} ${last_name}`);
       });
+
       cy.get("tr")
         .last()
         .children()
         .eq(-2)
+        .as("lastActive")
         .should("contain", year);
     });
 
     it.skip("should load the Audit log (Audit log should display views of dashboards)", () => {
```

Comment:
@flamber do you know/remember why this one is skipped?

Reply:
H2 problems EE#287